server/api/customers/customer.controller.js

```diff
@@ -0,0 +1,207 @@
+import CustomerService from './customer.service.js';
+// import WalletService from '../wallet/wallet.service.js';
```

Comment:
remove all commented code, development should not have this type of code anywhere